.github/workflows/daily.yml

```diff
@@ -1,12 +1,25 @@
-name: Daily
+name: Daily Tests
 on:
   schedule:
     - cron: "30 6 * * *"
   workflow_dispatch:
```

Comment:
please add a temporary "pull_request:" here to see if this workflow works.

Reply:
Given the `workflow_dispatch` trigger is enabled, we should be able to manually trigger the workflow without having to include the `pull_request` one. 

I suggest setting everything in place, and trying these out afterwards.

Reply:
I couldn't find a way to do that. Probably it's available only after the PR is merged.

Reply:
I've double checked the docs, and apparently that's allowed only when the workflow is in the `default` branch. I'll do so when we agree on the rest of the PR.

Reply:
We need to be sure this workflow is working properly.